_data/galleries/colorado-mountains.yml

```diff
@@ -0,0 +1,26 @@
+picture_path: colorado-mountains
+preview:
+  filename: Denver2019_Group.png
+  original: Denver2019_Group.png
+  thumbnail: Denver2019_Group.png
+pictures:
+- filename: Denver2019_Group.png
+  original: Denver2019_Group.png
+  thumbnail: Denver2019_Group.png
+  title: Colorado Mountains 2019 Group
+- filename: Denver2019_ValeDrone.png
+  original: Denver2019_ValeDrone.png
+  thumbnail: Denver2019_ValeDrone.png
+  title: Vale Frontside and Terrain Park 2019
```

Comment:
I think it's vail - we should change it here and in the filename

Reply:
Ah yeah, good catch